src/main/java/org/javacord/bot/commands/WikiCommand.java

```diff
@@ -38,7 +38,8 @@
      * @param args The command's arguments.
      * @throws IOException If the connection to the wiki failed.
      */
-    @Command(aliases = {"!wiki"}, async = true)
+    @Command(aliases = {"!wiki"}, async = true, usage = "!wiki [title|full] <search>",
+            description = "Shows a link to the Javacord wiki")
```

Comment:
Inconsistent with `!javadoc`. Basically they are the same. No argument => link, argument => search.
For the other it is `Can be used to search the JavaDocs`.
Actually all descriptions start with `Shows` except for `!javadoc`s.
Maybe `Shows a link to the JavaDoc or searches through it` and `Shows a link to the wiki or searches through it`?
afaik remember you shouldn't use the plural of `JavaDoc` and it is somewhat redundant to say `Javacord wiki` imho. :-)